openedx/adg/common/lib/mandrill_client/tasks.py

```diff
@@ -14,18 +14,21 @@
 
 
 @task()
-def task_send_mandrill_email(template, emails, context):
+def task_send_mandrill_email(template, emails, context, send_at=None):
     """
     Sends an email by calling send_mandrill_email, asynchronously
 
     Arguments:
         template (str): String containing template id
         emails (iterable): Email addresses of users
         context (dict): Dictionary containing email content
+        send_at (str): When this message should be sent as a UTC timestamp in YYYY-MM-DD HH:MM:SS format.
 
     Returns:
         None
     """
+    from openedx.adg.lms.webinars.helpers import save_scheduled_reminder_ids
```

Comment:
I think mandril client is generic for all sort of emails but there is some webinars related code in this file. can we move it webinars app?

Reply:
Actually its the generic task that sends all the emails, So, in this case we want to store ids that mandrill returns. I have added the helper in webinars app but when mandrill returns the response we need to pass that response to the called method `save_scheduled_reminder_ids `.

Reply:
Its like when we are sending reminder emails, just save the scheduled msg ids. All the save related logic is in webinars app.

Reply:
Let's review this code in a team meeting and see if we need improve this code in a followup story 👍